client-src/elements/chromedash-feature-detail.js

```diff
@@ -492,6 +499,51 @@ class ChromedashFeatureDetail extends LitElement {
     }
   }
 
+  hasStageActions(stage, feStage) {
+    // See if there is an API owners gate where actions are displayed.
+    const ownersGate = this.gates.filter(
```

Comment:
This could be a little simpler by using `this.gates.some(...)` to get a boolean instead of checking length below.